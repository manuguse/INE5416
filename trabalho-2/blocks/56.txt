1 1 10 10 10 10 10 16 25 25 25 25
1 1 11 10 10 16 16 16 16 25 31 25
1 6 11 11 11 16 20 21 21 27 27 27
2 6 6 6 3 16 21 21 21 27 32 32
3 3 3 3 3 17 21 23 23 28 28 32
4 4 12 12 3 17 17 23 23 23 26 26
4 7 13 14 14 17 17 23 26 26 26 33
4 4 13 13 14 14 14 24 24 24 24 33
4 4 8 8 9 18 18 22 24 29 29 29
5 8 8 9 9 18 22 22 24 29 29 30
5 5 9 9 15 19 19 22 24 30 30 30
5 9 9 15 15 15 19 19 19 30 30 34