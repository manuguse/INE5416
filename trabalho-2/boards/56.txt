5 - 5 3 7 4 - 4 - - 3 4
3 1 3 1 - 5 6 2 - 5 - -
- - - - 1 - - 6 - 3 - -
- - 1 - 6 - 5 3 - - - -
2 3 - 7 - 5 - 5 4 - - -
- 7 - - 1 - 4 - - - 3 4
- - 3 - - - - 2 - - - -
- 3 - - - 3 5 - - 1 7 -
- - - - - - - 4 - - 2 -
3 - 3 - - - - - - - - 6
- - 2 - - 3 - - - - 4 2
- 6 - - - 2 - 2 4 - - -