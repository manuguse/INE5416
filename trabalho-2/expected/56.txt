5 4 5 3 7 4 2 4 1 6 3 4
3 1 3 1 6 5 6 2 7 5 1 2
2 4 2 4 1 3 1 6 4 3 4 1
1 2 1 3 6 1 5 3 1 2 3 2
2 3 5 7 4 5 2 5 4 1 2 1
6 7 1 2 1 2 4 3 1 6 3 4
5 1 3 4 2 1 3 2 5 2 1 2
4 3 1 2 1 3 5 3 6 1 7 1
2 1 4 1 4 2 3 4 5 4 2 5
3 2 3 7 3 1 2 3 4 3 1 6
2 4 2 5 4 3 5 1 2 5 4 2
1 6 1 3 1 2 1 2 4 1 3 1